XouDevSpec/Classes/features/AnimeScreen/ViewModel/AnimeViewModel.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  AnimeViewModel.swift
+//  XouDevSpec
+//
+//  Created by Xander Schoeman on 2020/04/23.
+//
+
+import Foundation
+
+public protocol AnimeViewModelProtocol {
+    func getAnimeData(searchString: String, genreString: String)
+    func getTopAnimeData()
+}
+
+public class AnimeViewModel {
+    var anime = [AnimeDetails]()
+    var topAnime = [TopStruct]()
```

Comment:
You could actually remove this as wekk as the self?.anime and self?.topAnime in the ViewModel extensions because they serve no purpose. You would simply pass those values using view.animeRetrieveData and view.animateRetrieveTopData

Reply:
Noted. = )